tests/keras/backend/backend_test.py

```diff
@@ -362,8 +363,13 @@ def test_elementwise_operations(self):
         check_single_tensor_operation('mean', (4, 2, 3), WITH_NP, axis=-1, keepdims=True)
         check_single_tensor_operation('mean', (4, 2, 3), WITH_NP, axis=[1, -1])
 
+        check_single_tensor_operation('var', (4, 2), WITH_NP)
+        check_single_tensor_operation('var', (4, 2), WITH_NP, axis=1, keepdims=True)
+        # check_single_tensor_operation('var', (4, 2, 3), WITH_NP, axis=[1, -1])
```

Comment:
Please don't leave commented code (I know it happens in the codebase, but we'd like to avoid it).